cvat/apps/engine/serializers.py

```diff
@@ -2502,10 +2566,51 @@ def update(self, instance, validated_data):
 
             instance.project = project
 
+        if workspace_transferring:
+            if organization_id is not None:
+                try:
+                    organization_slug = list(
+                        Organization.objects.filter(pk=organization_id).values_list('slug', flat=True)
+                    )[0]
```

Comment:
```suggestion
                    organization_slug = (
                        Organization.objects.filter(pk=organization_id).values_list('slug', flat=True)
                    )[0]
```

Calling `list` is not needed here, the returned object allows indexing. Removing `list` will also result in a request with a `LIMIT 1` clause.